src/ORM/RulesChecker.php

```diff
@@ -298,8 +298,10 @@ public function checkDelete(EntityInterface $entity, array $options = [])
      * @param string $message The error message to show in case the rule does not pass.
      * @return callable
      */
-    public function isUnique(array $fields, $message = 'This value is already in use')
+    public function isUnique(array $fields, $message = null)
     {
+        $message = $message?: __n('cake', 'This value is already in use');
```

Comment:
There should be a space after `$message`. Also the function should be `__d` not `__n`.